test/function/samples/namespace-to-string-tag/main.js

```diff
@@ -0,0 +1,11 @@
+import * as ns from './foo.js';
+
+const { configurable, enumerable, value, writable } = Object.getOwnPropertyDescriptor(
+	ns,
+	Symbol.toStringTag
+);
```

Comment:
Why not explicitly check the `Object.assign` behaviour? Then this would also document why it is important and people do not need to Google it again.

Reply:
good idea, I'll add that to the test.